sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -66,8 +61,6 @@
 
   private static final String WORKER_THREAD_NAME =
       BatchSpanProcessor.class.getSimpleName() + "_WorkerThread";
-  private static final String SPAN_PROCESSOR_TYPE_LABEL = "spanProcessorType";
```

Comment:
We'll need to decide whether we expose a `Listener` interface to be able to push these metrics via an extension. Presumably after this PR.